### ISSUE TITLE
Could not decode rbs::Value into rbatis::rdbc::DateTime

### ISSUE THREAD

**Issue by longzou** [rbatis/rbatis#534]
match rbatis::decode::<rbatis::rbdc::DateTime>(rs.clone()) {
                Ok(rt) =>{
                    log::info!("decode datetime: {rt:?}");
                    Value::String(rt.to_string())
                },
                Err(err) => {
                    log::info!("decode datetime with {err}");
                    Value::String(rs.string())
                } ,
            }
it's could not decode into DateTime, below is the array message:

decode datetime with decode an not array value

**Comment by zhuxiujia** [comment]
`rs ` not a  array value

**Comment by longzou** [comment]
yes. rs is a string value. and the table column type is a datetime.
I think it should be decode as the value into really Type, isnt?

**Comment by zhuxiujia** [comment]
> yes. rs is a string value. and the table column type is a datetime. I think it should be decode as the value into really Type, isnt?

should be `[{"column":“2024-12-12 00:00:00Z”}]`   decode as the value into really Type,  `rbatis::decode::()` must be array value Because the database protocol is an array
